packages/tools/src/tools/displayTools/Contour/contourConfig.ts

```diff
@@ -2,13 +2,18 @@ import { ContourConfig } from '../../../types/ContourTypes';
 
 const defaultContourConfig: ContourConfig = {
   renderOutline: true,
+  outlineWidthAutoGenerated: 3,
```

Comment:
As requested, sets the defaults to the ones in the tools (mostly), with some auto generated enhancements.